app/new/page.tsx

```diff
@@ -1,14 +1,15 @@
-import { fetchSkills } from "../../api";
+import { fetchTitles, fetchSkills } from "../../api";
 import AddNewCvForm from "../../components/add_new_cv_form";
 import CVLayout from "../../components/layouts/cv";
 import { transformSkills } from "../../helpers";
 
 export default async function NewCv() {
   const skills = await fetchSkills();
+  const titles = (await fetchTitles()) || [];
```

Comment:
`transformSkills` ensures that `skills` prop is empty object or object with data - since we're not transforming `titles`, we're creating default value (empty array) here.